ci/build.py

```diff
@@ -284,8 +288,9 @@ def script_name() -> str:
                         default=1,
                         type=int)
 
-    parser.add_argument("-c", "--cache", action="store_true",
-                        help="Enable docker registry cache")
+    parser.add_argument("--no-cache", action="store_true",
```

Comment:
I find the name a bit confusing, maybe `--dockerhub--cache`?

Reply:
⚡️